### DISCUSSION TITLE
Google Pixel 7 and Pro - does Xdrip+ and Juggluco work on Pixel 7

### DISCUSSION THREAD

**Discussion by mrg9999** [NightscoutFoundation/xDrip#2419] [General]
Does  XDRIP+ app still work on Google Pixel 7?
I'd like a definitive answer rather than a simple Yes.

Is Xdrip+ a 64bit application as the Pixel 7 only supports 64 bit applications

Was it a simple install?

**Comment by rudysroost** [comment]
It does not work for me, nor does the BYOD Dexcom app. I'm running xDrip as
Dex Share Follower and I bought a BLU Advance L5
<https://www.amazon.com/gp/product/B07Z6Q9NCZ/> running Android 8.1 Oreo
(Go Edition) to use as a collector. It's working out, but I hope I don't
have to keep doing this for much longer.

I had a Pixel 5 before the Pixel 7, and after installing Android 13 on the Pixel 5, xDrip and the Dexcom app did not work on that phone either. I would get one reading after starting the phone and no more readings until the next time I re-started the phone.

**Reply by Navid200** [reply]
Please don't use email to post here.

**Reply by mrg9999** [reply]
>Please don't use email to post here.
What does that mean?

**Comment by Navid200** [comment]
> Does this app still work on Google Pixel 7? 

What app?

xDrip works on Pixel 7.

**Reply by mrg9999** [reply]
Ignoring - failed to read original message

**Comment by bryanw20** [comment]
Is it a 64bit app? I don't know.

Does it load via apk on a 6a or 7pro. Yes it does.

Does it occasionally (maybe one a week) just miss signal for 30-45min. Yes it does, but comes back, or a reboot restores the connection.

Can you upload a backup to Google drive cloud, then restore it, yes it also works. 
6a to 6a was quick & easy, 6a to 7p had to do manual file to restore from as it wasn't looking for a 6a file folder.
